### PR TITLE
Add force-aligned feature

### PR DESCRIPTION
**Pull request by Lukinos**
> 'force aligned' attribute will align automatically all attributes based on the position of the first attribute.

As I said (https://github.com/beautify-web/js-beautify/pull/936#issuecomment-244077843), I have some issues to how correctly test this feature.

~~Edit: I will fix the issue about startsWith ASAP.~~

### CODE REVIEW

File: test/data/html/tests.js [hunk 1]
```diff
@@ -300,6 +300,12 @@ exports.test_data = {
             ],
             indent_attr: ' ',
             indent_over80: ' '
+        }, {
+            options: [
+                { name: "wrap_attributes", value: "'force-aligned'" }
+            ],
+            indent_attr: '\\n     ', /* Number of spaces should be dynamic */
```
Comment (bitwiseman) [inline test/data/html/tests.js]: add more matrix vars to match the dynamic size and adjust the tests?  Specifically consider changing the elements used for the tests to ones that will align in the same position.  Or one of a couple positions...

File: js/lib/beautify-html.js [hunk 2]
```diff
@@ -396,16 +401,21 @@
                     if (content.length && content[content.length - 1] !== '=' && input_char !== '>' && space) {
                         //no space after = or before >
                         var wrapped = this.space_or_wrap(content);
-                        var indentAttrs = wrapped && input_char !== '/' && wrap_attributes !== 'force';
+                        var indentAttrs = wrapped && input_char !== '/' && !this.startWidth(wrap_attributes, 'force');
```
Comment (bitwiseman) [inline js/lib/beautify-html.js]: You don't want to be doing this search every time. Do it once early on and save it in var.

**Comment by Lukinos** [comment]
I am not sure why it fails. Is the problem on my end ?

**Comment by bitwiseman** [comment]
Remember to run `./build`and commit the fixed code.

**Comment by bitwiseman** [comment]
So this closes #916, right?

Could you squash all these to one commit and rebase it on the beautify-web/js-beautify master branch?

**Comment by Lukinos** [comment]
Hopefully, I fulfill correctly your request about squashing and rebasing. 

Yes. If it is possible, add a reference to the ticket #455 also. When I search on my beloved search engine, I was redirected to this issue first.

**Comment by bitwiseman** [comment]
squash and rebase: yes, you did that correctly. 
"add a reference to the ticket": you just did. 😄 

Thanks for your contribution!

**Pull request merged**